### ISSUE TITLE
Numpy 1.14

### ISSUE THREAD

**Issue by tkittel** [mctools/mcpl#34]
The MCPL python interface tests ends with a new error:
```
ValueError: Changing the dtype to a subarray type is only supported if the total itemsize is unchanged
```
This seems to be caused by a change introduced with numpy 1.14: https://stackoverflow.com/questions/48267058/using-np-view-with-changes-to-structured-arrays-in-numpy-1-14

**Comment by tkittel** [comment]
Seems to be a bug in numpy 1.14, the question is of course if this will be fixed in 1.15

https://stackoverflow.com/questions/48267058/using-np-view-with-changes-to-structured-arrays-in-numpy-1-14

**Comment by tkittel** [comment]
MCPL v1.2.1 was finally released, fixing this.
